packages/web-runtime/src/store/user.js

```diff
@@ -42,6 +42,9 @@ const actions = {
       dispatch('hideModal')
       dispatch('loadSettingsValues')
 
+      // Reset store
+      this.reset()
```

Comment:
To be blunt, I don't understand where the `reset` is coming from. 🤔 at least it's not something that `vuex` provides.
What you could use is `replaceState`, but that's probably too much for what you want to achieve.
There are some actions dispatched in the lines above. Maybe it makes more sense for the time being to use those for resetting state. For example the `SET_USER` mutation doesn't reset the quota, you could fix that. And a new action in the `navigation` store module for clearing the dynamic nav items would help as well.

Reply:
The reset function is part of the functionality of vuex-extensions library that is used in web for example to use createStore. It uses replaceState: https://github.com/huybuidac/vuex-extensions/blob/c80fda5e0f1bed3362cc2064086499d2b2e80fa5/src/index.js#L18

I thought, resetting the whole store state might prevent possible future bugs by login switches between different account types. 

If parts of store need to be keeped on logout it's possible to do partial reset of selected children/grandchildren, for example cleanUpLoginState could be implemented like this
`this.reset({ 
  self: false,
  nested: false,
  modules: { user: { self: true} }
})`
instead of manually cleaning user, his capabilities and userReady.

What do you think of it?

Reply:
Hm... found the issue now why resetting the whole store is not going to work. The `navigation` module has dynamic and static nav items. The `static` nav items are pushed into the store during bootstrap. But the bootstrap doesn't run again after login. Maybe that should. 😅 We can look into that during our beta phase.

I like your approach of only resetting particular modules. For the user it definitely makes sense like you proposed it.
For solving another issue that you had after relogin you need to reset the dynamic nav items only from the `navigation` store in the runtime.

Reply:
For the record, this is what happens on re-login: tries to access on of the static nav items and since they are cleared this results in an exception.
<img width="720" alt="Screenshot 2022-04-14 at 09 41 13" src="https://user-images.githubusercontent.com/3532843/163338979-908b910b-5cf6-43e8-acd6-893cab0770b8.png">